README.md

```diff
@@ -149,8 +149,156 @@ df[["name", "review_count"]]
 
 #### [Spotify](./spotify) -- Collect Albums, Artists, and Tracks Metadata
 
+<details>
+  <summary>How many followers does Eminem have?</summary>
+  
+```python
+from dataprep.connector import connect
+
+# You can get ”spotify_client_id“ and "spotify_client_secret" by registering as a developer https://developer.spotify.com/dashboard/#
+conn_spotify = connect("spotify", _auth={"client_id":spotify_client_id, "client_secret":spotify_client_secret}, _concurrency=3)
+
+df = await conn_spotify.query("artist", q="Eminem", _count=500)
+
+df.loc[df['# followers'].idxmax(), '# followers']
```

Comment:
What is .idxmax() ?

Reply:
.idxmax() will search for the item in the Pandas series, df['# followers'], with the highest number. It will return the index for this item.